open_spiel/game_parameters.cc

```diff
@@ -55,8 +57,24 @@ std::string GameParameter::ToString() const {
   switch (type_) {
     case Type::kInt:
       return absl::StrCat(int_value());
-    case Type::kDouble:
-      return absl::StrCat(double_value());
+    case Type::kDouble: {
+      // XXX(maintainers): We make sure to include a '.0' for doubles that hold
+      //      integer values to avoid deserializing them as integers (see
+      //      implementation in `GameParameterFromString`).
+      //
+      //      See also: #382.
+      //
+      //
+      // We cannot use StrCat as that would default to exponential notation
+      // sometimes. For example, the default format of 10^-9 is the string
+      // "1e-9". For that reason, we use std::stringstream here.
+      //
+      // FIXME(maintainers): Avoid hardcoding the number of digits after the
```

Comment:
Can you change this to TODO: Avoid hardcoding...  

It causes a problem and hence blocks the import. If you have it anywhere else, can you do the same? Thanks.

Reply:
To be clear, the problem is the FIXME. It needs to be TODO. No need to have anyone in parentheses.

Reply:
Ah that's cool that you have some `FIXME` hooks :-) I will move that `FIXME` to a `TODO` and remove the (maintainers) in a minute. I will keep the `XXX`s in place though without changing them to `TODO`s, since they are more like important considerations than stuff that needs to be done.